Cesium3DTilesSelection/src/TilesetLoadTilesetDotJson.cpp

```diff
@@ -37,9 +37,15 @@ struct Tileset::LoadTilesetDotJson::Private {
       const std::shared_ptr<spdlog::logger>& pLogger,
       bool useWaterMask);
 
+  static void workerThreadLoadTileContext(
```

Comment:
This function is no longer purely synchronous, because it sometimes need to load parent layer.jsons. So it should return a future, as should `workerThreadHandleResponse`.

Reply:
As currently written, we'll start loading tiles before we know the chain of underlying layers, which can cause buggy, non-deterministic loading behavior depending on the timing of responses.